addons/packages/fluxcd-helm-controller/metadata.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: helm-controller.fluxcd.community.tanzu.vmware.com
+spec:
+  displayName: "Flux Helm Controller"
+  longDescription: "The Helm Controller is a Kubernetes operator, allowing one to declaratively manage Helm chart releases with Kubernetes manifests."
+  shortDescription: "The Helm Controller is a Kubernetes operator, allowing one to declaratively manage Helm chart releases with Kubernetes manifests."
```

Comment:
long and short descriptions are the same. We should change the short description.

Reply:
fixed